packages/react/src/AsgardeoReactClient.ts

```diff
@@ -56,7 +56,7 @@ import AuthAPI from './__temp__/api';
 import getMeOrganizations from './api/getMeOrganizations';
 import getScim2Me from './api/getScim2Me';
 import getSchemas from './api/getSchemas';
-import {AsgardeoReactConfig} from './models/config';
+import { AsgardeoReactConfig } from './models/config';
```

Comment:
```suggestion
import {AsgardeoReactConfig} from './models/config';
```

Looks like this have been formatted by the IDE, let's fix these